benchmarks/benchmarks/stats.py

```diff
@@ -20,3 +22,60 @@ def time_anderson_ksamp(self):
         with warnings.catch_warnings():
             warnings.simplefilter('ignore', UserWarning)
             anderson_ksamp(self.rand)
+
+def random(size):
+    return rand(*size)
```

Comment:
I forgot the syntax of ``np.random.rand`` to take multiple dimensions and hence did unpacking. Will use
  
    np.random.rand(2,2)